Wobble.Extended/HotReload/HotLoaderGame.cs

```diff
@@ -19,7 +19,7 @@ public abstract class HotLoaderGame : WobbleGame
         /// </summary>
         public HotLoaderScreen HotLoaderScreen { get; protected set; }
 
-        public HotLoaderGame(HotLoader hl)
+        public HotLoaderGame(HotLoader hl, bool preferWayland) : base(preferWayland)
```

Comment:
Make this argument optional. Shouldn't be required to specify this.

```suggestion
        public HotLoaderGame(HotLoader hl, bool preferWayland = false) : base(preferWayland)
```